### DISCUSSION TITLE
Using Quarto as the core piece of aggregator for reproducible research "paper"

### DISCUSSION THREAD

**Discussion by rgaiacs** [quarto-dev/quarto-cli#6240] [Show and Tell]
### Description

Hi,

I'm working on the idea of a open source file format agnostic aggregator for [reproducible](https://the-turing-way.netlify.app/reproducible-research/overview/overview-definitions.html) research "paper" based on Quarto.

![concept drawio](https://github.com/quarto-dev/quarto-cli/assets/1506457/04f08746-2ea1-42a4-a0ca-3f79289cb640)

Researchers will write using their favourite tool (Quarto, Jupyter Notebook, ...) and version control their work in their favourite Git host (GitHub, GitLab, Bitbucket, ...). The aggregator operator creates containers capable of reproduce the code blocks in the documents, renders the document to Markdown (adding relevant metadata) using Quarto, and stores the Markdown in a central location. After process all the documents, the aggregator operator render all the documents into a website using Quarto.

# User Case

- new independent academic journals (think of the next [The Journal of Open Source Software](https://joss.theoj.org/)
- research group/department website
- (add your here)

# Limitations

- Reproduce some steps in high performance computing fields like astronomy and genetics is inviable

# Why do you use Markdown as intermediate step?

1. Markdown is a cache for all code execution in a document.
2. Markdown can be stored in a Git host and "adequately" render (yes, I know that GitHub does **not** render YAML header)
3. Markdown is supported by many static website generators and can be integrated in existing websites

**Comment by schochastics** [comment]
Hi,
I am also part of this project and I just want to give some more non-technical background. We at [GESIS](https://gesis.org)  are currently building up a service portfolio for social scientists to work with digital behavior data (DBD). This specific service here is an open science/source platform where users get easy access to method implementations, tutorials and documentations. This platform is built on the shoulders of Quarto because, among other things, the platform is supposed to be language-agnostic (we do not focus on just R or just Python). 

Content creators can "submit" their content (tutorials, notebooks, scripts, packages,...) in any format they are comfortable with (ipynb, rmd qmd, docx, .R,.py,docx) and our engine build on top of quarto does the conversion to markdown and then html to make it available to the end-user via the rendered website.  

We committed to Quarto because we are convinced of its philosophy. Its a great tool for reproducible research and it is so adaptable to the diverse environments users (specifically in the interdisciplinary field of computational social science)  work in.
